api/v1alpha1/storageclient_types.go

```diff
@@ -52,6 +52,9 @@ type StorageClientStatus struct {
 
 	// ConsumerID will hold the identity of this cluster inside the attached provider cluster
 	ConsumerID string `json:"id,omitempty"`
+
+	// ProviderState will hold the hash of provider state of client
+	ProviderState string `json:"providerState,omitempty"`
```

Comment:
`ProviderState` is the wrong none to use here, everything captured inside that hash is about consumer configuration not provider state. Also, a timestamp would really help in debugging


```
ConsumerConfig: 
    hash: <the  hash>
    timestamp: <the last time the hash changed>
```

@leelavg @rchikatw I don't really like the term `ConsumerConfig` so if you can come up with an alternative name it will be great!